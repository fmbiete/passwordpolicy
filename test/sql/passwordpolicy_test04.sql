DROP USER IF EXISTS test_pass;

CREATE USER test_pass WITH PASSWORD 'password';

DROP USER IF EXISTS test_pass;
