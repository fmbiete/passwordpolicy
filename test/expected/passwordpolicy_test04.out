DROP USER IF EXISTS test_pass;
NOTICE:  role "test_pass" does not exist, skipping
CREATE USER test_pass WITH PASSWORD 'password';
ERROR:  password is easily cracked.
DROP USER IF EXISTS test_pass;
NOTICE:  role "test_pass" does not exist, skipping
